### ISSUE TITLE
Unable to CustomerService.get without customer ID

### ISSUE THREAD

**Issue by ondrejbartas** [googleads/google-api-ads-ruby#54]
My customer are connecting to my application by oauth authorization.

When I get refresh token and access token I was setted up Adwords::Api instance withou client_customer_id and asked CustomerService.get for authorized customer. Adwords API has in documentation:
```
CLIENT_CUSTOMER_ID_IS_REQUIRED
Client Customer Id is required if CustomerIdMode is set to CLIENT_EXTERNAL_CUSTOMER_ID. Starting version V201409 ClientCustomerId will be required for all requests except for CustomerService.get
```

so it should not be problem to do this request. But when you try to initialize instance of Adwords:API without client_customer_id you get:
```
AdwordsApi::Errors::BadCredentialsError: Missing client customer ID for the request.
from /Users/bartas/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/google-adwords-api-0.15.0/lib/adwords_api/credential_handler.rb:74:in `validate_headers_for_server'
```

Do you have any idea how to get customer id of authorized user from api by giving only access token?

Bes Ondrej

**Comment by ondrejbartas** [comment]
https://cloud.githubusercontent.com/assets/45809/7979341/a42c3574-0a9e-11e5-8c60-b0b63a84fab4.png

I managed to create same request by Postman without client customer id to v201502 so only problem is that Adwords::API is throwing exception when it should not throw.

**Comment by mcloonan** [comment]
This should be fixed now; please update to version 0.15.1 of the adwords gem.

**Comment by ondrejbartas** [comment]
Thank you very much. Now it is working perfectly :D
